<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MouseDebug</class>
 <widget class="QMainWindow" name="MouseDebug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>发送设置</string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ReportID：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="rpIDLineEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>发送字节数：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sendBytesLineEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>数据发送窗口</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="clearRevDataBtn">
         <property name="text">
          <string>清空接收区</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="clearSendDataBtn">
         <property name="text">
          <string>清空发送区</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QTextEdit" name="sendDataTextEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="sendDataBtn">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>300</height>
       </size>
      </property>
      <property name="title">
       <string>Display 参数</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Manufacturer：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Product：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Release：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Interface：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Vendor_ID：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Product_ID：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>usage_page：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>usage：</string>
       </property>
      </widget>
      <widget class="QLabel" name="manufacturerText">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="productText">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="releaseNumText">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="interfaceNumText">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="VIDText">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="PIDText">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="usagePageText">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>210</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="usageText">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>240</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>数据接收窗口</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTextEdit" name="recvDataTextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string>USB 配置</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>HID 设备</string>
       </property>
      </widget>
      <widget class="QPushButton" name="on_offBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboHIDBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>VID：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>PID:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="VIDLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="PIDLineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
